www/layouts/partials/home_course_cards.html

```diff
@@ -1,7 +1,8 @@
 <div class="new-courses course-cards standard-width container mx-auto mt-3">
 {{ $studioBaseUrl := getenv "OCW_STUDIO_BASE_URL" }}
-{{ if $studioBaseUrl }}
-{{  with getJSON (print (strings.TrimSuffix "/" $studioBaseUrl) "/api/websites/?type=course")  }}
+{{ $courseStarterSlug := getenv "OCW_IMPORT_STARTER_SLUG" }}
+{{ if and $studioBaseUrl $courseStarterSlug }}
+{{  with getJSON (print (strings.TrimSuffix "/" $studioBaseUrl) "/api/websites/?type=" $courseStarterSlug)  }}
```

Comment:
Can we use [querify](https://gohugo.io/functions/querify/) here to ensure the course slug is escaped properly?